ResearchKit/Common/ORKResult.m

```diff
@@ -1670,6 +1671,65 @@ - (ORKStepResult *)stepResultForStepIdentifier:(NSString *)stepIdentifier {
 
 @end
 
+@implementation ORKLocationQuestionResult
+
+- (void)encodeWithCoder:(NSCoder *)aCoder {
+    [super encodeWithCoder:aCoder];
+    ORK_ENCODE_OBJ(aCoder, locationAnswer);
+}
+
+- (instancetype)initWithCoder:(NSCoder *)aDecoder {
+    self = [super initWithCoder:aDecoder];
+    if (self) {
+        ORK_DECODE_OBJ_CLASS(aDecoder, locationAnswer, NSValue);
+    }
+    return self;
+}
+
++ (BOOL)supportsSecureCoding {
+    return YES;
+}
+
+- (BOOL)isEqual:(id)object {
+    BOOL isParentSame = [super isEqual:object];
+    
+    __typeof(self) castObject = object;
+    return (isParentSame &&
+            //ORKEqualObjects(_locationAnswer, castObject.locationAnswer)) ;
```

Comment:
This seems to extend comparison of ORKResult by adding location coordinate match.